.github/workflows/labels.yml

```diff
@@ -0,0 +1,14 @@
+name: "Label PR"
+
+on:
+  pull_request_target:
+
+jobs:
+  labels:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'NixOS'
```

Comment:
Does this mean it'll only labels PRs which are made from branches within the NixOS organization?

Reply:
No, it just blocks the action from running in peoples forks.